setup.py

```diff
@@ -0,0 +1,20 @@
+import os
+from setuptools import setup
+
+setup(
```

Comment:
As a matter of fact I don't think there is a use case for using setuptools, which is meant to make it easy to import the module as a library, while are not at all in the business of providing this app as an installable, let alone a standalone library. I think this is not part of the goal of "configuring the app in view of deploying it and testing it in production".

Reply:
I disagree. For building new containers automatically, there is a quite simple workflow on OBS:
1. get the current sources and build an RPM
2. once the RPM is built, the container is rebuilt using the new package.
By using setuptools, this is a really simple procedure.
All we need is this part in our .spec file:
```specfile
%build
%python_build

%install
%python_install
%python_expand %fdupes %{buildroot}%{$python_sitelib}
```

Reply:
Note to myself: Comment this